doc/modules/mod_revproxy.md

```diff
@@ -33,23 +33,36 @@ For more details about the listeners configuration please take a look at [ejabbe
 
 #### Configuring routes
 
-To define reverse proxy rules, add the following entry to the modules section.
+To define reverse proxy rules, add entries defining routes to `modules.mod_revproxy.routes`.
 
-```Erlang
-{mod_revproxy,
-   [{routes, [{"www.erlang-solutions.com", "/admin", "_",
-               "https://www.erlang-solutions.com/"},
-              {":var.com", "/:var", "_", "http://localhost:8080/"},
-              {":domain.com", "/", "_", "http://localhost:8080/:domain"}]
-     }]},
-```
+#### `modules.mod_revproxy.routes.host`
+* **Syntax:** non-empty string
+* **Default:** no default
```

Comment:
Is it possible to be `non-empty` and `no default` at the same time?

Reply:
I think so, when the key has to be a non-empty string, but it wouldn't make any sense to provide any default. Basically when a key is mandatory.